docs/src/pages/discover-more/showcase/appList.js

```diff
@@ -609,6 +609,18 @@ const appList = [
     stars: 4,
     dateAdded: '2020-01-31',
   },
+  {
+    title: 'A Clone of Google Keep',
+    description:
+      'A twin-looking Google Keep clone with light/dark Material theme as adapted exactly in Google Keep,' +
+      ' with a GraphQL backend written in Golang',
+    image: 'googlekeepclone.jpg',
```

Comment:
Do you think that you could update the image to match the other images format and increase the image quality a bit?

Reply:
I scaled the image down to `900x675` and optimized the JPEG slightly. How should this be done? Any suggestions? I have the original image though